File: tests/fixtures/compare/sample.erl
-module(sample).
-export([process/1, empty/0]).

%% TODO: add proper error handling

process(Data) ->
    case Data of
        _ -> ok;
        {error, Reason} -> {error, Reason};
        {ok, Value} -> Value
    end.

empty() -> ok.

risky(X) ->
    exit(X).

File: tests/fixtures/compare/sample.lua
-- TODO: add error handling
globalVar = 42

function process(items)
    for i, item in ipairs(items) do
        print(item)
    end
    os.execute("ls -la")
    loadstring("return 1")()
end

function empty()
end

function complex(a, b, c, d, e)
    return a + b + c + d + e
end
